Cannot run the slick-grid react app

### Describe the bug

Encountered the following error about regenerator-runtime while trying to run the application both in [Slick Grid React Demo](https://github.com/ghiscoding/slickgrid-react-demos) and in a fresh new React project using the [Quick Start Tutorial](https://ghiscoding.gitbook.io/slickgrid-react/getting-started/quick-start):

```
> slickgrid-react@4.0.0 dev
> vite


  VITE v7.1.6  ready in 582 ms

  ➜  Local:   http://localhost:8080/
  ➜  press h + enter to show help
✘ [ERROR] Could not resolve "regenerator-runtime/runtime.js"

    node_modules/slickgrid-react/dist/index.js:1:7:
      1 │ import 'regenerator-runtime/runtime.js';
        ╵        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "regenerator-runtime/runtime.js" as external to exclude it from the bundle,
  which will remove this error and leave the unresolved path in the bundle.

/Users/furquan/Documents/slickgrid-react-demos/bootstrap5-i18n-demo/node_modules/esbuild/lib/main.js:1477
  let error = new Error(text);
              ^

Error: Error during dependency optimization:

✘ [ERROR] Could not resolve "regenerator-runtime/runtime.js"

    node_modules/slickgrid-react/dist/index.js:1:7:
      1 │ import 'regenerator-runtime/runtime.js';
        ╵        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "regenerator-runtime/runtime.js" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.


    at failureErrorWithLog (/Users/furquan/Documents/slickgrid-react-demos/bootstrap5-i18n-demo/node_modules/esbuild/lib/main.js:1477:15)
    at /Users/furquan/Documents/slickgrid-react-demos/bootstrap5-i18n-demo/node_modules/esbuild/lib/main.js:946:25
    at /Users/furquan/Documents/slickgrid-react-demos/bootstrap5-i18n-demo/node_modules/esbuild/lib/main.js:1355:9
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5) {
  errors: [Getter/Setter],
  warnings: [Getter/Setter]
}

Node.js v23.7.0
```

**Points to note:**

- A colleague of mine was able to run the [Slick Grid React Demo](https://github.com/ghiscoding/slickgrid-react-demos) with no issues, his node version is `v22.20.0`. 
- Another thing to note is that, when I tried to run the Slick Grid React Demo using `yarn` and not using `npm`, it works fine and does not give any error. But for a standalone fresh project, both `yarn` and `npm` gives the above error.


**Package versions:**
- npm: `10.9.2`
- Node: `23.7.0`
- yarn: `1.22.22`

### Reproduction

- Clone the [Slick Grid React Demo](https://github.com/ghiscoding/slickgrid-react-demos)
- Go inside the demo folder: `cd slickgrid-react-demos/bootstrap5-i18n-demo`
- Install dependencies: `npm i`
- Run the app: `npm run dev`

or

- Create a fresh react project (I used vite)
- Follow the [Quick Start Tutorial](https://ghiscoding.gitbook.io/slickgrid-react/getting-started/quick-start) of Slick Grid React
- Run the app: `npm run dev`

### Which Framework are you using?

React

### Environment Info

```shell
| Executable          | Version |
| ------------------- | ------- |
| React               | 19.1.1 |
| Slickgrid-Universal | 9.9.0 |
| TypeScript          | 5.9.2 |
| Browser(s)          | 141.0.7390.108 |
| System OS           | macOS 15.6.1 |
```

### Validations

- [x] Follow our [Code of Conduct](https://github.com/ghiscoding/slickgrid-universal/blob/master/.github/CODE_OF_CONDUCT.md)
- [x] Read the [Wikis](https://github.com/ghiscoding/slickgrid-universal/wiki).
- [x] Check that there isn't [already an issue](https://github.com/ghiscoding/slickgrid-universal/issues) that reports the same bug to avoid creating a duplicate.
- [x] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/ghiscoding/slickgrid-universal/discussions).
- [x] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

--- a/demos/react/package.json
+++ b/demos/react/package.json
@@ -66,7 +66,6 @@
     "cypress": "catalog:",
     "cypress-real-events": "catalog:",
     "globals": "catalog:",
-    "regenerator-runtime": "^0.14.1",
     "rxjs": "catalog:",
     "sass": "catalog:",
     "typescript": "catalog:",

--- a/frameworks/slickgrid-react/package.json
+++ b/frameworks/slickgrid-react/package.json
@@ -100,7 +100,6 @@
     "react": "^19.2.0",
     "react-dom": "^19.2.0",
     "react-router-dom": "^7.9.4",
-    "regenerator-runtime": "^0.14.1",
     "remove-glob": "catalog:",
     "rxjs": "catalog:",
     "sass": "catalog:",

--- a/frameworks/slickgrid-react/src/index.ts
+++ b/frameworks/slickgrid-react/src/index.ts
@@ -1,4 +1,3 @@
-import 'regenerator-runtime/runtime.js';
 import { SlickgridReact } from './components/slickgrid-react.js';
 import { SlickRowDetailView } from './extensions/slickRowDetailView.js';
 import type { GridOption, RowDetailView, SlickgridReactComponentOutput, SlickgridReactInstance } from './models/index.js';

